packages/signals_core/lib/src/async/async_signal.dart

```diff
@@ -12,11 +12,12 @@ typedef AsyncSignalErrorBuilder<R> = R Function(Object? error);
 /// Builder on a [AsyncSignal] callback
 typedef AsyncSignalBuilder<R> = R Function();
 
-/// A compound [Signal] that wraps a [Stream] or [Future]
+/// A [Signal] that stores value in [AsyncState]
 class AsyncSignal<T> extends ValueSignal<AsyncState<T>> {
```

Comment:
What's the use case. Let's say I combine it with a FutureSignal?

Reply:
Here is an example I had in mind:
https://github.com/rodydavis/signals.dart/discussions/120